packages/eslint-plugin/src/rules/no-empty-interface.ts

```diff
@@ -1,4 +1,9 @@
 import * as util from '../util';
+import { AST_NODE_TYPES } from '@typescript-eslint/experimental-utils';
+import {
+  RuleFix,
+  RuleFixer,
+} from '@typescript-eslint/experimental-utils/dist/ts-eslint';
```

Comment:
Prefer importing the namespace instead of deep importing from the `dist` of another package.

```suggestion
import { AST_NODE_TYPES, TSESLint  } from '@typescript-eslint/experimental-utils';
```